roles/openshift_facts/library/openshift_facts.py

```diff
@@ -490,7 +490,7 @@ def set_selectors(facts):
         facts['hosted']['metrics'] = {}
     if 'selector' not in facts['hosted']['metrics'] or facts['hosted']['metrics']['selector'] in [None, 'None']:
         facts['hosted']['metrics']['selector'] = None
-    if 'logging' not in facts:
+    if 'logging' not in facts or not isinstance(facts['logging'], dict):
```

Comment:
I'd prefer to have a function near `get_local_facts_from_file` that fixes these right after they're read if we can swing it.

Reply:
I'm hopeful we can do away with the file in it's entirety.  This seems to affect 3.6->3.7 somehow, or installing a new 3.7 cluster with some stale data lying around.

I think we should probably just do a PR against 3.7 instead.